resources/splunk_app.rb

```diff
@@ -33,11 +31,11 @@ def app_path
   end
 
   def config_scope
-    'local'
+    @config_scope ||= (resource_name == :splunk_app_custom ? 'default' : 'local')
   end
 
-  def splunk_app_path
-    Pathname.new(install_dir).join('etc/apps')
+  def app_cache_path
+    @app_cache_path ||= Pathname.new(Chef::Config['file_cache_path']).join('splunk_ingredient/old_apps')
```

Comment:
I guess I am unsure of how you are differentiating "simple" references vs others as it seems to me that consistency should be a goal.